stripe/__init__.py

```diff
@@ -32,12 +32,6 @@
 
 # API resources
 from stripe.api_resources import *  # noqa
-from stripe.api_resources import checkout  # noqa
-from stripe.api_resources import issuing  # noqa
-from stripe.api_resources import radar  # noqa
-from stripe.api_resources import reporting  # noqa
-from stripe.api_resources import sigma  # noqa
-from stripe.api_resources import terminal  # noqa
```

Comment:
I think this is okay, but can you check that after importing the library, something like `stripe.issuing.Card` still works?

Reply:
it does!

Reply:
👍 Thanks!